server/index.js

```diff
@@ -2,8 +2,8 @@
 
 const express = require('express');
 const logger = require('./logger');
-
-const argv = require('minimist')(process.argv.slice(2));
+const two = 2;
```

Comment:
`const arguementsIndex = 2;`

the purpose of this number is to pickup any arguments when running the file.
e.g. anything that's not `node server/index.js`
https://www.npmjs.com/package/minimist

f.x. one can overwrite the port by doing `node server/index.js -port 8008` and argv will consist of `{port: '8008'}`

The reason for slicing is because, of course there is no interest in `node server/index.js`